sw/device/tests/clkmgr_jitter.c

```diff
@@ -0,0 +1,34 @@
+// Copyright lowRISC contributors.
```

Comment:
Okay to keep, but `chip_csr_rw` test already covers this.

Reply:
Sorry I take that back - only the coverage of TL ports of all blocks is enabled for the `chip_csr_rw` test. It will not record the coverage of `jitter_en_o` output of clkmgr all the way to AST. Lets keep this test.